Gio-2.0.gir

```diff
@@ -6930,6 +6943,19 @@ calling g_printerr() in the invoking process.</doc>
           </parameter>
         </parameters>
       </method>
+      <method name="printerr_literal" c:identifier="g_application_command_line_printerr_literal">
```

Comment:
I guess for all the GLib things we just wait until GObject-Introspection is properly merged in there and no separate syncing of introspection data is needed.

@alatiera The GNOME SDK is going to build GLib like that then, right?

Reply:
Ok, so either way just needs a bit of waiting here